s3/pkg/datastore/yig/storage/storage.go

```diff
@@ -40,6 +41,14 @@ var (
 	RootContext = context.Background()
 )
 
+func (yig *YigStorage) BucketDelete(ctx context.Context, in *pb.Bucket) error {
+	panic("implement me")
```

Comment:
In other non-implemented functions.. are we returning 'nil'  .. Please cross check and  ensure consistency..

Reply:
Comments can be implemented in next PR  , should not block merge.